src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,25 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
-    userCountries.forEach(function(countryName) {
-        var countryTotal = userData[countryName].length;
-
-        countriesArr.push([countryName, countryTotal]);
-        total += countryTotal;
-    });
-
+    var countryTotal = 0;
+    var date = userData.lastUpdated;
+    for (key in userData.institutes) {
```

Comment:
@AnaghHegde, take a closer look at the object I used that operation on. Are you sure the object I was targeting only has 2 keys?